.gitattributes

```diff
@@ -0,0 +1 @@
+test/* linguist-documentation
```

Comment:
Calling it `linguist-documentation` doesn't seem right. Is there a better (not perl) option?

Reply:
`github-linguist` only offers a few options for manually overriding misinterpreted languages. I will change this in a commit later to `*.t -linguist-detectable`, which will make the `*.t` files undetectable by linguist.

Reply:
Ohhhh, that's my fault. I thought you were saying this was "documentation for linguists", like it was somehow related to translations. Now I see linguist is the library and you're calling these files documentation. That'd be fine with me, as is your other option. Totally up to youl